### PR TITLE
Add quota to cs usage

### PR DESCRIPTION
**Pull request by nsinkov**
## Changes proposed in this PR

Add quota information to cs usage command

## Why are we making these changes?

Quota shows the max resources a user will get. What they see now is non-preemptible share which is often bigger than their max quota, so they can get confused when they run out of quota and don't realize it.

### CODE REVIEW

**Comment by nsinkov** [comment]
Does this need its own version bump?

**Comment by dposada** [comment]
@nsinkov Yes, we will want another bump for this

**Comment by dposada** [comment]
@nsinkov Can you resolve conflicts?

**Comment by nsinkov** [comment]
@dposada resolved

**Comment by nsinkov** [comment]
@dposada go, go, go?

File: cli/cook/version.py [hunk 1]
```diff
@@ -1 +1 @@
-VERSION = '2.12.0'
+VERSION = '2.12.1'
```
Comment (dposada) [inline cli/cook/version.py]: How about:

```suggestion
VERSION = '2.13.0'
```

File: cli/cook/subcommands/usage.py [hunk 2]
```diff
@@ -167,17 +167,20 @@ def print_formatted_cluster_or_pool_usage(cluster_or_pool, cluster_or_pool_usage
...
+         'N/A'],
         ['Current Usage',
          usage_map['cpus'],
          format_job_memory(usage_map),
-         usage_map['gpus']]
+         usage_map['gpus'],
+         usage_map['jobs']]
     ]
-    print_info(tabulate(rows, headers=['', 'CPUs', 'Memory', 'GPUs'], tablefmt='fancy_grid'))
+    print_info(tabulate(rows, headers=['', 'CPUs', 'Memory', 'GPUs', 'Jobs'], tablefmt='fancy_grid'))
```
[the remark this answers is not in the document]
Comment (nsinkov) [reply cli/cook/subcommands/usage.py]: "bytes" is in the humanfriendly library. So will leave as-is for now

**Pull request merged**